R/utils.R

```diff
@@ -206,6 +206,10 @@ phantom_paths <- function() {
   } else if (is_osx()) {
     path <- '~/Library/Application Support'
     path <- if (dir_exists(path)) file.path(path, 'PhantomJS')
+  } else if (is_linux()) {
+    path <- '~/bin'
+    path <- if (!dir_exists(path)) dir.create(path,recursive = TRUE)
```

Comment:
I think this isn't the right place to create the directory -- this function simply returns a possible path for the executable to go; it should not create the path.

The suggestion in #74 seems like the right way to go.